geotrek/altimetry/tests/test_elevation.py

```diff
@@ -305,9 +310,10 @@ class SamplingTestPath(TestCase):
     model = Path
     step = settings.ALTIMETRIC_PROFILE_PRECISION
 
-    def setUp(self):
-        if self.model is None:
-            SkipTest()
+    @classmethod
+    def setUpTestData(cls):
+        if cls.model is None:
+            SkipTest(reason="None")
```

Comment:
J'expliciterais plus que "None"
```suggestion
            SkipTest(reason="No model")
```